include/envoy/tcp/conn_pool_nats.h

```diff
@@ -10,7 +10,7 @@
 
 namespace Envoy {
 namespace Tcp {
-namespace ConnPool {
+namespace ConnPoolNats {
```

Comment:
Nit: It might be enough to change the filename without having to change the namespace. Modifying the namespace to `Tcp::ConnPoolNats` might cause some confusion with `Nats::ConnPool`.